docs/tutorials/ethereum-local-development.md

```diff
@@ -0,0 +1,264 @@
+
+![Server configuration](</img/ganache-create-workspace-2.png>)
+
+:::info
+**Note:** It is not mandatory to change `PORT NUMBER` or `NETWORK ID`, but they are set to the same value as Hardhat, so the rest of the Tutorial will look the same.
+
+The `AUTOMINE` option is disabled, same as in the *Hardhat* section of this tutorial, for debugging purposes. For more information, consult the [official documentation](https://trufflesuite.com/docs/ganache/reference/cli-options/#miner).
+:::
+
+#### 3. Deploy smart contract
+
+Deploy the sample smart contract by launching the following command in a console window:
+
+```bash
+truffle migrate
+```
+
+:::warning
+**Note:** Make sure the `development` configuration, in the `networks` section of `truffle-config.js` is uncommented, and has these values (which should be the same as the previous step).
+
+```
+    development: {
+     host: "127.0.0.1",     // Localhost (default: none)
+     port: 8545,            // Standard Ethereum port (default: none)
+     network_id: "1337",       // Any network (default: none)
+    },
+```
+:::
+
+Log information similar to this should be displayed:
+
+```bash
+   Deploying 'MetaCoin'
+   --------------------
+   > transaction hash:    0xcdb820827306ebad7c6905d750d07536c3db93f4ef76fd777180bdac16eaa2ca
+   > Blocks: 1            Seconds: 4
+   > contract address:    0xd095211a90268241D75919f12397b389b1062f6F
+   > block number:        329
+   > block timestamp:     1673277461
+   > account:             0x131D37F433BAf649111278c1d6E59843fFB26D28
+   > balance:             99.98855876
+   > gas used:            414494 (0x6531e)
+   > gas price:           20 gwei
+   > value sent:          0 ETH
+   > total cost:          0.00828988 ETH
+
+   > Saving artifacts
+   -------------------------------------
+   > Total cost:          0.01144124 ETH
+```
+
+Take note of the contract address, **you'll need it later**.
+
+### Archive Docker image
+
+In order to index the local Ethereum node, we need to use a pre-built Docker image of Subsquid's Ethereum Archive. You can do it, by creating a `docker-compose.archive.yml` file, and paste this:
+
+```yaml
+version: "3"
+
+services:
+  worker:
+    image: massimocodes/eth-archive-worker:latest
+    environment:
+      RUST_LOG: "info"
+    ports:
+      - 8080:8080
+    command: [
+            "/eth/eth-archive-worker",
+            "--server-addr", "0.0.0.0:8080",
+            "--db-path", "/data/db",
+            "--data-path", "/data/parquet/files",
+            "--request-timeout-secs", "300",
+            "--connect-timeout-ms", "1000",
+            "--block-batch-size", "10",
+            "--http-req-concurrency", "10",
+            "--best-block-offset", "10",
+            "--rpc-urls", "http://host.docker.internal:8545/",
+            "--max-resp-body-size", "30",
+            "--resp-time-limit", "5000",
+            "--query-concurrency", "16",
+    ]
+    volumes:
+      - db:/data/db
+      - parquet:/data/parquet
```

Comment:
I think Ozgur has prepared a version when no parquet is needed

Reply:
It shouldn't be needed, but the archive won't start without the option `"--data-path"`:

```bash
ethereum-local-indexing-worker-1  | error: The following required arguments were not provided:
ethereum-local-indexing-worker-1  |   --data-path <DATA_PATH>
ethereum-local-indexing-worker-1  | 
ethereum-local-indexing-worker-1  | Usage: eth-archive-worker --db-path <DB_PATH> --data-path <DATA_PATH> --request-timeout-secs <REQUEST_TIMEOUT_SECS> --connect-timeout-ms <CONNECT_TIMEOUT_MS> --block-batch-size <BLOCK_BATCH_SIZE> --http-req-concurrency <HTTP_REQ_CONCURRENCY> --max-resp-body-size <MAX_RESP_BODY_SIZE> --resp-time-limit <RESP_TIME_LIMIT> --server-addr <SERVER_ADDR> --best-block-offset <BEST_BLOCK_OFFSET> --rpc-urls <RPC_URLS> --query-concurrency <QUERY_CONCURRENCY>
ethereum-local-indexing-worker-1  | 
ethereum-local-indexing-worker-1  | For more information try '--help'
ethereum-local-indexing-worker-1 exited with code 2
```

Reply:
@ozgrakkurt was this is fixed in the latest release?

Reply:
Don't know if you already had a conversation on another channel (Slack), but I'm quoting my own conversation with Ozgur:

> we would have to go to old code branch modify it and then release it for this. Current head is not very usable at the moment

So I guess it is fixed, but in a different branch, so we don't have an image ready, yet.

Reply:
Fixed with latest release